core/chains/starknet/chain.go

```diff
@@ -129,10 +129,8 @@ func (c *chain) Ready() error {
 	return c.StartStopOnce.Ready()
 }
 
-func (c *chain) Healthy() error {
-	return c.StartStopOnce.Healthy()
-}
-
 func (c *chain) HealthReport() map[string]error {
-	return map[string]error{c.Name(): c.Healthy()}
+	report := map[string]error{c.Name(): c.StartStopOnce.Healthy()}
+	utils.MergeMaps(report, c.txm.HealthReport())
```

Comment:
Is this custom func more efficient than using https://pkg.go.dev/golang.org/x/exp/maps#Copy?

Reply:
It would be nice to reuse a familiar function instead of introducing a subtly different one.
There shouldn't normally be duplicate keys anyways, right?